src/assets/style/module/dialog/_dialog.less

```
@@ -0,0 +1,161 @@
+.dialog {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+    // LESS itself tries to evaluate calc(), using ~"" here to prevent that
+    max-height: ~"calc(100% - 2*36px - 2*7px)";
+    width: 600px;
+    //width: fit-content;
+    margin: auto;
+    padding: 0px 10px 10px 10px;
+    position: absolute;
+    top: 36px;
+    left: 0;
+    right: 0;
+    background-color: @dark-06;
+    color: @light-gray;
+    font-size: 12px;
+    border: 6px solid @blue-medium;
+    border-radius: 8px;
+    box-sizing: border-box;
+    box-shadow: 0 0 50px -10px rgba(0,0,0,.3);
+    opacity: 0;
+    transition: opacity 0.5s ease;
+    pointer-events: none;
+    z-index: 200;
+}
+
+.dialog h1,
+.dialog h2,
+.dialog h3,
+.dialog h4,
+.dialog h5,
+.dialog h6 {
+    font-size: 16px;
+    padding-top: 15px;
+}
+
+.dialog li {
+    padding: 15px 0 0 0;
+}
+
+.dialog li + li {
+    padding: 5px 0;
+}
+
+ .dialog a {
+    color: @light-gray;
+}
+
+.dialog table,
+.dialog th,
+.dialog td {
+    border: 1px solid #666666;
+    border-collapse: collapse;
+}
+
+.dialog th {
+    padding: 5px;
+}
+
+.dialog td {
+    padding: 3px;
+}
+
+.dialog.open {
+    opacity: 1;
+    pointer-events: auto;
+}
+
+.dialog-title {
```

```suggestion
.dialog-bd {
```
I think this is the convention now?